content/en/docs/reference/command-line-tools-reference/feature-gates/persistent-volume-last-phase-transition-time.md

```diff
@@ -13,6 +13,11 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.29"
+    toVersion: "1.29"
+  - stage: stable
+    defaultValue: true
+    fromVersion: "1.31"
+    toVersion: "1.33"
```

Comment:
Filling `toVersion` is better when the feature gate is removed from k/k. @RomanBednar

Reply:
Ack, removed.